packages/SwingSet/src/kernel/kernel.js

```diff
@@ -263,6 +278,29 @@ export default function buildKernel(kernelEndowments) {
     }
   }
 
+  async function deliverAndLogToVat(vatID, kernelDelivery, vatDelivery) {
```

Comment:
This name is really confusing.  My first thought was that it should be `deliverToVatAndLog` but "log" is also a noun so it's still ambiguous.  Really what you mean (I think) is `deliverToVatThenLog` but that's still a little ambiguous and anyway is not a name anyone should ever use.  Or you could say `deliverToVatAndThenLogDelivery` but now we're into COBOL levels of foolishness.  Perhaps another approach is to declare that relevant logging is implicit in everything we do and so just call it `deliverToVat`.  But the bottom line is this needs a better name.

Reply:
Yeah, as mentioned in the commit comment, these three functions (deliverToVat, processNotify, and deliverAndLogToVat) want to have a name refactoring. I think this particular function wants to be named `deliverToVat`, because that's exacly what it's doing. The previous deliverToVat is only dealing with message sends (not notifies), so it wants a name that includes "message" or "send" or something.

I think a deeper issue is that I've been inconsistent (and drifting) around my naming for these delivery things. I started out using "dispatch" as the verb/noun, as in `dispatch.deliver` and `dispatch.notifyFulfillToPresence`, but 1: it's harder to say than just "deliver", and 2: as this kernel-to-vat invocation moves to being more about sending a data object from kernel to vatManager to vatWorker (and less about invoking some particular method of a `dispatch` object, which now happens inside the vatWorker, maybe inside a different process altogether, and generally invisible to the kernel), I want a name for the data object. "Dispatch" is both a verb and a noun (as in "dispatches from the eastern front"), but "delivery" has the benefit of being obviously not a verb. It may help to hash out a new naming system for these pieces, and then rename all the functions to match.